README.md

```diff
@@ -38,7 +47,9 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+``In JavaScript, scope is the set of variables, objects, and functions you have access to. There's local scope and global scope; local scope refers to the context in which variables objects and functions are used within a function. This is implemented by 'this' and 'bind'. Global scope refers to the use of objects variables and functions within the entire script (out side of the functions).
+
+I think.
```

Comment:
Overall pretty close! Everything you said is right, the only exception is that `this` and `bind` don't really have to do with scope. `this` and `bind` are what we call 'context', which is related to when functions are called. 

Scope is more general, and as you said, it's just which variables and functions you can use at any point in your program.